RESOURCES.md

```diff
@@ -5,25 +5,79 @@ mathsteps@socratic.org
 
 ### Existing open source step-by-step solvers
 
-[MathPiper](mathpiper.org) by [Ted Kosan](https://github.com/tkosan)
+- [PRESS PRolog Equation Solving System](https://github.com/maths/PRESS)
```

Comment:
is this a step-by-step solver? if not, we should probably make a separate section for CAS (though that could end up pretty big cause there's a bunch of those out there)